mkosi/run.py

```diff
@@ -381,7 +367,10 @@ def chroot_cmd(root: Path, *, options: Sequence[PathString] = (), network: bool
     else:
         cmdline += ["--unshare-net"]
 
-    return cmdline
+    # No exec here because we need to clean up the /work directory afterwards.
+    cmdline += ["sh", "-c", f"$0 \"$@\" && rm -rf {root / 'work'} || exit $?"]
```

Comment:
Isn't the `|| exit $?` superfluous here, i.e. wouldn't
```suggestion
    cmdline += ["sh", "-c", f"$0 \"$@\" && rm -rf {root / 'work'}"]
```
do the same thing (as long as there is no command afterwards)? If the command before the `&&` exits nonzero this will trigger the `||` and exit with the exit code code the command before the `&&`, but without the `||`, the right side of `&&` will not be triggered either and the compound command will have the return value of left hand side (i.e. the command before `&&`), no?